addons/mass_mailing/static/src/js/wysiwyg.js

```diff
@@ -20,6 +20,18 @@ const MassMailingWysiwyg = Wysiwyg.extend({
         return res;
     },
 
+    /**
+     * @override
+     */
+     setValue: function (currentValue) {
+        const initialDropZone = this.$editable[0].querySelector('.o_editable');
+        const currentDropZone = (new DOMParser().parseFromString(currentValue, "text/html")).querySelector('.o_editable');
+        if (initialDropZone && !currentDropZone) {
+            initialDropZone.innerHTML = currentValue;
```

Comment:
Since you are parsing the html from string already, I suggest to store the result into a variable, then check the querySelector inside the `if` condition (no local variable for that), and then use `replaceChildren` instead of `innerHTML` so that the string html is parsed only once.

Reply:
@dmo-odoo Done changes.

Reply:
@dhba-odoo Thanks ! 👍  Note that there is also the other comment that needs addressing.